.github/workflows/check.yaml

```diff
@@ -10,28 +10,25 @@ name: R-CMD-check
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
-  #cancel-in-progress: true
+  cancel-in-progress: true
 
 jobs:
   R-CMD-check:
     runs-on: ${{ matrix.config.os }}
-
     name: ${{ matrix.config.os }} (${{ matrix.config.r }})
-
     strategy:
       fail-fast: false
       matrix:
         config:
-          #- { os: macos-latest, r: "release" }
-          - { os: windows-latest, r: "release", rtools-version: "42" }
-          - { os: windows-latest, r: "release", rtools-version: "" }
+          - { os: macos-latest, r: "devel" }
+          - { os: macos-latest, r: "release" }
+          - { os: macos-latest, r: "oldrel-1" }
           - { os: windows-latest, r: "devel", rtools-version: "43" }
-          - { os: windows-latest, r: "devel", rtools-version: "" }
+          - { os: windows-latest, r: "release", rtools-version: "42" }
           - { os: windows-latest, r: "oldrel-1", rtools-version: "42" }
```

Comment:
Is `rtools-version` needed here?

Reply:
Currently if not stated r-lib/setup-r will prefer to install rtools40. extendr requires rtools42 as we use a linker first included from there. [I did some testing/derping and also r-lib team advice us to specify rtools version for now](https://github.com/r-lib/actions/issues/720#issuecomment-1518393895)

When rtools42 or above becomes the default we can remove the version numbers. It is often (not always) possible to use a previous rtools.version to compile the next R.version. I predict 30% risk next R/rtools release will require these numbers updated :)

Reply:
Hmmm...I have updated prqlr's workflow today and it seems rtools43 was set uped automatically.

https://github.com/eitsupi/prqlr/actions/runs/4771440538/jobs/8483371510#step:4:1

```log
Run r-lib/actions/setup-r@v2
D:\a\_temp\R-4.3.0-win.exe /VERYSILENT /SUPPRESSMSGBOXES /DIR=C:\R
Downloading https://github.com/r-hub/rtools43/releases/download/latest/rtools43.exe...
D:\a\_temp\rtools43.exe /VERYSILENT /SUPPRESSMSGBOXES
C:\ProgramData\Chocolatey\bin\choco.exe install ghostscript qpdf --no-progress
```

Reply:
In short, I suspect that the `rtools-version` is no longer needed for `release` version, since the release version of Windows has already switched to R 4.3.